lib/mariaex/connection/ssl.ex

```diff
@@ -1,31 +1,8 @@
 defmodule Mariaex.Connection.Ssl do
-
   def recv(sock, bytes, timeout), do: :ssl.recv(sock, bytes, timeout)
 
-  def recv_active(sock, timeout, buffer \\ :active_once) do
-    receive do
-      {:ssl, ^sock, buffer} ->
-        {:ok, buffer}
-      {:ssl_closed, ^sock} ->
-        {:disconnect, {tag(), "async_recv", :closed, buffer}}
-      {:ssl_error, ^sock, reason} ->
-        {:disconnect, {tag(), "async_recv", reason, buffer}}
-    after
-      timeout ->
-        {:ok, <<>>}
-    end
-  end
-
   def tag(), do: :ssl
 
-  def fake_message(sock, buffer), do: {:ssl, sock, buffer}
```

Comment:
My understanding here was that this was used to "flush" out anything that was currently in the socket buffer. Because that code path is gone, I do not believe there is any further use for this.